addons/l10n_es/taxes_common.xml

```diff
@@ -35,272 +35,557 @@
 
         -->
         <!-- Account Tax Tags -->
-        <record id="tax_tag_1" model="account.account.tag">
-            <field name="name">4% IVA Soportado (operaciones corrientes)</field>
+        <record id="tag_es_01" model="account.account.tag">
+            <field name="name">IVA devengado. Base imponible</field>
```

Comment:
I don't see these tags anymore relevant, as they keep the old taxes codes scheme (but without parents). Isn't better to reformat them? cc @oco-odoo